modules/guestbook/guestbook-service/src/main/java/br/com/objective/training/search/GuestbookModelDocumentContributor.java

```
@@ -42,19 +44,20 @@ public void contribute(Document document, Guestbook guestbook) {
         /**
          * @lmarqs: The code below has been copied from GuestbookIndexer#doGetDocument.
          */
+        try {
+            document.addDate(Field.MODIFIED_DATE, guestbook.getModifiedDate());
 
-        document.addDate(Field.MODIFIED_DATE, guestbook.getModifiedDate());
-
-        Locale defaultLocale = null;
+            Locale defaultLocale = defaultLocale = PortalUtil.getSiteDefaultLocale(guestbook.getGroupId());
+            String localizedField = LocalizationUtil.getLocalizedName(Field.TITLE, defaultLocale.toString());
 
-        try {
-            defaultLocale = PortalUtil.getSiteDefaultLocale(guestbook.getGroupId());
-        } catch (PortalException e) {
-            throw new Exception(e);
+            document.addText(localizedField, guestbook.getName());
+        } catch (PortalException pe) {
```

Should we throw this exception?